pyproject.toml

```diff
@@ -87,34 +87,11 @@ docs = [
     "furo>=2022.3.4",
 ]
 dev = [
-    # tf
-    "tensorflow>=2.9.0,<3.0.0",  # cf. https://github.com/mindee/doctr/issues/454
-    "tensorflow-addons>=0.17.1",
-    "tf2onnx>=1.9.2",
-    # torch
-    "torch>=1.8.0",
-    "torchvision>=0.9.0",
-    # test
-    "pytest>=5.3.2",
-    "coverage[toml]>=4.5.4",
-    "hdf5storage>=0.1.18",
-    "onnxruntime>=1.11.0",
-    "requests>=2.20.0",
-    # quality
-    "flake8>=3.9.0",
-    "isort>=5.7.0",
-    "black>=22.1",
-    "mypy>=0.812",
-    "pydocstyle[toml]>=6.1.1",
-    # docs
-    "sphinx>=3.0.0,!=3.5.0",
-    "sphinxemoji>=0.1.8",
-    "sphinx-copybutton>=0.3.1",
-    "docutils<0.18",
-    "recommonmark>=0.7.1",
-    "sphinx-markdown-tables>=0.0.15",
-    "sphinx-tabs>=3.3.0",
-    "furo>=2022.3.4",
+    "python-doctr[tf]",
+    "python-doctr[torch]",
+    "python-doctr[testing]",
+    "python-doctr[quality]",
+    "python-doctr[docs]"
```

Comment:
That's dangerous because it will install the last published release. But in dev mode, you'll want to have the version of which the code is on your laptop

Reply:
@frgfm that's a really good catch ... do you know an alternative way without rewriting all deps ?

Reply:
Nope that's why I rewrite all deps :sweat_smile:

Reply:
(this is not a dynamic build system yet, so that's not possible by design to the best of my knowledge)

Reply:
would you like to open a PR for this ? :sweat_smile: Unfortunately im blocked until Wednesday (and the free time i have is currently filled with implementing ViT and ViTSTR in doctr :sweat_smile: )